From 2084430a8caf17a80300b5be35843300db60a185 Mon Sep 17 00:00:00 2001
From: Bo Shen <voice.shen@atmel.com>
Date: Thu, 8 Jan 2015 14:11:07 +0800
Subject: [PATCH] ARM: atmel: sama5d4 xplained: fix the LCD parameters

Correct the LCD pixel clock, remove unused vsync parameter,
and also correct the include file.

Signed-off-by: Bo Shen <voice.shen@atmel.com>
---
 board/atmel/sama5d4_xplained/sama5d4_xplained.c | 5 ++---
 1 file changed, 2 insertions(+), 3 deletions(-)

diff --git a/board/atmel/sama5d4_xplained/sama5d4_xplained.c b/board/atmel/sama5d4_xplained/sama5d4_xplained.c
index 4c6b8fc..311dd4e 100644
--- a/board/atmel/sama5d4_xplained/sama5d4_xplained.c
+++ b/board/atmel/sama5d4_xplained/sama5d4_xplained.c
@@ -15,7 +15,7 @@
 #include <asm/arch/clk.h>
 #include <asm/arch/sama5d3_smc.h>
 #include <asm/arch/sama5d4.h>
-#include <atmel_lcdc.h>
+#include <atmel_hlcdc.h>
 #include <atmel_mci.h>
 #include <lcd.h>
 #include <mmc.h>
@@ -110,8 +110,7 @@ static void sama5d4_xplained_usb_hw_init(void)
 vidinfo_t panel_info = {
 	.vl_col = 480,
 	.vl_row = 272,
-	.vl_clk = 9000,
-	.vl_sync = ATMEL_LCDC_INVLINE_NORMAL | ATMEL_LCDC_INVFRAME_NORMAL,
+	.vl_clk = 9000000,
 	.vl_bpix = LCD_BPP,
 	.vl_bpox = LCD_OUTPUT_BPP,
 	.vl_tft = 1,
-- 
1.9.1

